An error on `linux-64`:
```
  config = pluginmanager.hook.pytest_cmdline_parse(
ImportError while loading conftest '/croot/wandb_1738577440349/test_tmp/tests/conftest.py'.

tests/conftest.py:21: in <module>

    import git  # noqa: E402

../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python3.10/site-packages/git/__init__.py:298: in <module>

    raise ImportError("Failed to initialize: {0}".format(_exc)) from _exc

E   ImportError: Failed to initialize: Bad git executable.
```


Try with `export GIT_PYTHON_REFRESH=quiet` in the test environment

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -11,7 +11,8 @@ source:
 
 build:
   number: 0
-  skip: true  # [py<38]
+  # skip: true  # [py<38]
+  skip: true  # [py>39]  # temporarily for rdkit, which requires 3.10+
   # skipping s390x as we do not have setproctitle package for this arch. Also, the current package 
   # will go to Snowflake-specific channel only, so there is no urgency to build it on s390x.
   skip: true  # [linux and s390x]

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -12,7 +12,7 @@ source:
 build:
   number: 0
   # skip: true  # [py<38]
-  skip: true  # [py>39]  # temporarily for rdkit, which requires 3.10+
+  skip: true  # [py<310]  # temporarily for rdkit, which requires 3.10+
   # skipping s390x as we do not have setproctitle package for this arch. Also, the current package 
   # will go to Snowflake-specific channel only, so there is no urgency to build it on s390x.
   skip: true  # [linux and s390x]
@@ -143,7 +143,7 @@ test:
     # ERROR tests/unit_tests/test_lib/test_hashutil.py::test_md5_file_hashes_on_mounted_filesystem[wandb_core-1kB]
     # ERROR tests/unit_tests/test_lib/test_hashutil.py::test_md5_file_hashes_on_mounted_filesystem[wandb_core-50kB]    
 
-    # - python tools/local_wandb_server.py start &
+    - export GIT_PYTHON_REFRESH=quiet
     - pytest tests -v
 
   requires:
@@ -169,7 +169,7 @@ test:
     - boto3
     - jax
     - openai
-    - tensorflow
+    #DDD - tensorflow
     - gymnasium
     - torcheval  # for torch
     - tensorboard

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -143,6 +143,8 @@ test:
     # ERROR tests/unit_tests/test_lib/test_hashutil.py::test_md5_file_hashes_on_mounted_filesystem[wandb_core-1kB]
     # ERROR tests/unit_tests/test_lib/test_hashutil.py::test_md5_file_hashes_on_mounted_filesystem[wandb_core-50kB]    
 
+    - rm tests/unit_tests/test_cli.py  # using docker
+
     - export GIT_PYTHON_REFRESH=quiet
     - pytest tests -v
 

